php_phongo.c

```diff
@@ -673,8 +673,9 @@ bool phongo_execute_bulk_write(mongoc_client_t* client, const char* namespace, p
 
 	/* The Write failed */
 	if (!success) {
-		if (error.domain == MONGOC_ERROR_SERVER || error.domain == MONGOC_ERROR_WRITE_CONCERN) {
-			zend_throw_exception(php_phongo_bulkwriteexception_ce, error.message, error.code TSRMLS_CC);
+		if ((error.domain == MONGOC_ERROR_COMMAND && error.code != MONGOC_ERROR_COMMAND_INVALID_ARG) ||
+			error.domain == MONGOC_ERROR_SERVER || error.domain == MONGOC_ERROR_WRITE_CONCERN) {
+			phongo_throw_exception(PHONGO_ERROR_WRITE_FAILED TSRMLS_CC, "%s", error.message);
```

Comment:
The changes to these three lines look to have been introduced by https://github.com/mongodb/mongo-php-driver/pull/781/commits/373d2f26d1249c81caa67d84e7ba40a8de87dd2e. Should these be removed?

IIRC, @derickr caught this in one of his earlier reviews, when the PR was originally changing `phongo_execute_bulk_write()` instead of `phongo_execute_command()`.

As-is, this reverts changes from https://github.com/mongodb/mongo-php-driver/commit/110d13e4b3532354d128101b1bc701a3fadf6889 and https://github.com/mongodb/mongo-php-driver/commit/8e0e836913a6c826246585c43d64e6a8f82d444f.

Reply:
On a semi-related note: [PHPC-1086](https://jira.mongodb.org/browse/PHPC-1086).